velox/dwio/parquet/tests/reader/ParquetReaderBenchmark.cpp

```diff
@@ -211,3 +366,563 @@ int main(int argc, char** argv) {
   folly::runBenchmarks();
   return 0;
 }
+
+#if 0
```

Comment:
Can we use `/* */` instead of `#if 0` since these are not code?  Some static analytics tools will try to parse the code.

Reply:
Fixed. This PR also applies to https://github.com/facebookincubator/velox/pull/3211 Can you please also take a look at that one? Thanks